rpm_spec/core-agent.spec.in

```diff
@@ -373,6 +374,19 @@ switching from user to root. Since all the user data in a VM is accessible
 already from normal user account, there is not much more to guard there. Qubes
 VMs are single user systems.
 
+%package dracut
+Summary:        Dracut configuration needed in Qubes OS
+License:        GPL
+Group:          Qubes
+Requires:       dracut
+Requires:       qubes-core-agent = %{version}
+
+%description dracut
+Dracut configuration needed in Qubes OS.  This enables
+passwordless login in rescue and emergency modes, as well
+as configuring dracut to only install certain drivers and
+omit others.
```

Comment:
There is already a bunch of dracut modules in qubes-kernel-vm-support package (linux-utils repo). I'm not suggesting moving it there (but if you want, that is an option too), but to better describe what is in this one, and how is it different from the other one.